static/css/styles.css

```diff
@@ -0,0 +1,4 @@
+.sidebar_list {

```

Comment:
Add here `ul.sidebar_list` to indicate that it should work with unordered lists.